service/static/js-src/muk.funder.js

```diff
@@ -3571,10 +3573,18 @@ $.extend(edges, {
             }
         };
         this.synchronise = function() {
+            if(initialPageLoad && this.field == "record.rioxxterms:project.funder_name.exact") {
+                initialPageLoad = false
+                var funderList = this.edge.result.data.aggregations.oahybrid.buckets["0"].funder.buckets;
+                for (var i = 0; i < 10; i++) {
+                    this.selectTerm(funderList[i].key)
```

Comment:
Why '10' and not some other number what if there are more/fewer funders in the list?